dev-db/influxdb/files/influxdb.init.d

```diff
@@ -0,0 +1,30 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
```

Comment:
We no longer do CVS, so please strip those headers. Also repoman warns about it, so please upgrade your repoman, or start using it. [applies to all files]